recipes/tiatoolbox/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "tiatoolbox" %}
+{% set version = "1.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tiatoolbox-{{ version }}.tar.gz
+  sha256: 0d864b3fadc078ddf7982003977f711ac245e7d1973662849eb248365174bdd4
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - tiatoolbox=tiatoolbox.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.7
+    - opencv >=4.5.4
+  run:
+    - albumentations >0.5.0
+    - click >=7.0
+    - defusedxml
+    - glymur
+    - imagecodecs
+    - matplotlib-base
+    - numpy
+    - opencv >=4.5.4
+    - openslide-python ==1.1.2
```

These hard constrains are usually a bad idea. Is this the only version guaranteed to work with this package?

---

Unfortunately, yes. This is defined in the package itself.

---

yes this is the latest version of openslide-python. The previous versions break on latest versions of python.

---

In that case I'd recommend open upper bounds:

```suggestion
    - openslide-python >=1.1.2
```

---

that should be fine